### DISCUSSION TITLE
(Info): Categorical Features Discussion

### DISCUSSION THREAD

**Discussion by aayu5hgit** [aayu5hgit/histograde#4] [General]
![image](https://github.com/aayu5hgit/histograde/assets/86314754/d11c7f9b-ce73-45de-aea0-b56550846f15)

Basically, there are two major types of `feature categorization` that needs to be work upon. Let's dive into it and learn more.

```mermaid
graph TD;
    A[Feature Types]-->B[Architectural Features];
    A-->C[Cytological Features];
```

 here we will be discussing about their sub-types → Get basic knowledge → Sorting the features from easy to difficult so as to start working.

**Comment by aayu5hgit** [comment]
## 🟢 Architectural Features
```mermaid
graph TD;
    A[Architectural Features];
    A-->C[Loss of Stratification];
    A-->D[Loss of Polarity];
    A-->E[Drop Shape Rete Ridges];
    A-->F[Keratin Pearls];
    A-->G[Loss of Cohesion];
```

### ⭕ **Loss of Stratification**
It refers to a **disruption or flattening** of the normal layers (strata) of cells in a tissue. In many tissues, cells are organized in layers like bricks in a wall. When this organization becomes disordered, it can indicate a problem.
To identify the presence of Loss of Stratification in a cell, search for specific signs:
- Lack of uniform shape
- Absence of horizontal layer
- Occurs from lower region (upto middle region)
- [Example Image](https://github.com/aayu5hgit/histograde/assets/86314754/5c48d471-d94d-4e10-8a06-9f2c537c91ca)

### ⭕ **Loss of Polarity**
It refers to a situation where cells within a tissue lose their normal organization and orientation. In simpler terms, it means that the cells are no longer maintaining their organized structure and arrangement. They might become disoriented and jumbled.
- Less binding of cells
- Disorganized Arrangement
- [Example Image](https://www.nature.com/articles/aps201120/figures/1)

### ⭕ **Keratin Pearls**
These structures are made up of tightly packed, concentric layers of keratin, a tough protein.
Keratin pearls look like small, round structures within the tissue. They often have a central core that is denser and harder.
- Clusters of cell keratinization 
- [Example Image](https://media.sciencephoto.com/c0/51/01/32/c0510132-800px-wm.jpg)

**Reply by anujbagad** [reply]
### ⭕ **Loss of Cohesion**
It refers to the **weakening of the normal connections** between **adjacent cells** within a tissue layer. 
Epithelial tissues are made up of **closely packed cells** that form protective layers on body surfaces and line internal organs.
But, when there is a **loss of cohesion**, these cells **might not stick together** properly, which can lead to various issues.
In cancer, loss of cohesion might be a sign of aggressive behavior or metastasis (spread).
To identify, look for the following:
- Gaps Between Cells
- Disordered Arrangement
- Loosely connected or not connected
- [Example Image](https://www.researchgate.net/publication/348663503/figure/fig4/AS:982554526097418@1611270662751/Hobnail-loss-of-cellular-polarity-loss-of-cellular-cohesion-variant-of-PTC-Piling-or.jpg)

**Comment by aayu5hgit** [comment]
## 🟢 Cytological Features
```mermaid
graph TD;
    A[Cytological Features];
    A-->B[Atypical Mitotic Figures];
    A-->C[Increased No. of Nucleoli];
    A-->D[Apoptotic Mitoses];
    A-->E[Single Cell Keratinization];
```

### ⭕ **Single Cell Keratinization**
It refers to a process where an individual cell starts producing a substance called keratin, which is a tough and protective protein. This process is not supposed to happen in certain types of cells, and when it does, it can indicate an abnormality.

To identify the presence of single cell keratinization in a cell, search for specific signs that indicate the abnormal keratin production:
- Dark pink hue in cytoplasm
- Nucleus is not visible clearly
-  Cells might look different from their normal counterparts; They might have an irregular shape.
- [Example Image](https://d45jl3w9libvn.cloudfront.net/jaypee/static/books/9789351524410/Chapters/images/80-1.jpg)

### ⭕ **Increased Number of Nucleoli**
When we talk about an **"increased number of nucleoli"** in epithelium, we're referring to a situation where there are more nucleoli than usual within the nuclei of the cells that make up an epithelial tissue layer. This can be an indicator of certain cellular activities or abnormalities.
To identify, look for the following:
- **2 or more nucleoli** in a cell
- Larger Size
- Variability (some will have 2 or more nucleoli, some will not)
- [Example Image](https://github.com/aayu5hgit/histograde/assets/86314754/63a1ae25-6372-44ab-87ed-1cf2b742b953)
